File: templates/template.js
const fs = require("node:fs");
let input;
try {
    input = fs.readFileSync("input.txt", "utf8");
} catch {
    input = fs.readFileSync("2025/dayX/test.txt", "utf8");
}
input = input.split("\n");
input = input.map((x) => x.replace("\r", "")).filter((x) => x);

function main(input_data) {
    // write program here
    // input_data = list of string, one element per line of raw input
    return null;
}

const result = main(input);
console.log(result);

File: templates/template.py
input_data = []
try:
    with open("input.txt") as file:
        for line in file:
            line = line.strip()
            if line:
                input_data.append(line)
except FileNotFoundError:
    with open("2025/dayX/input.txt") as file:
        for line in file:
            line = line.strip()
            if line:
                input_data.append(line)


def main(input_data):
    # write program here
    # input_data: list[str], one element per line of raw input
    return None


result = main(input_data)
print(result)
